controller/rancher/rancher_cert_dir_test.go

```diff
@@ -87,6 +87,22 @@ func TestCheckCertDirUpdate(t *testing.T) {
 		t.Fatalf("Error building config %v", err)
 	}
 
+	if len(newConfigs[0].Certs) != 3 {
```

Comment:
This asserts that the incomplete cert (that we just added) was not included in the Certs[] generated for the LB. Since both fullchain.pem and privkey.pem were not present, this cert should be skipped.